dimagi/pages/templates/sections/commcare/product/design_tool.html

```diff
@@ -0,0 +1,84 @@
+{% load i18n %}
+{% load dimagidotcom %}
+
+<section class="advisory border-top"
+         id="advisory">
+  <div class="content pad-vert-medium">
+    <h2 class="content-width-large blue gap-top-line align-center">
+      {% blocktrans %}
+        Build Customizable Frontline Solutions for Any Team
+      {% endblocktrans %}
+    </h2>
+    <p class="content-width-large lead-text pad-bottom-small align-center">
+      {% blocktrans %}
+        CommCare allows you to build custom solutions without compromising 
+        on the unique needs of your team. <br class="hide-until-medium" />
+        Whether you’re evaluating a program or supporting frontline 
+        workers, CommCare can help.
+      {% endblocktrans %}
+    </p>
+
+    <div class="design-tool content grid-3-at-medium gap-bottom-small">
+      <a class="tools-cards border-rounded-large column border-all align-center pad-horz-small-at-medium pad-top-small accent-gray-bg pad-bottom-small"
+         href="{% url 'monitoring_evaluation' %}">
+        <div class="theme-icon gap-bottom-line align-center">
+          {% include 'svg/commcare/icon/monitor_teams.html' %}
+        </div>
+        <h5 class="theme-color gap-bottom-line">
+          {% blocktrans %}
+            M&E Teams
+          {% endblocktrans %}
+        </h5>
+        <p class="body-color font-weight-light">
+          {% blocktrans %}
+            Enhance your last-mile data collection and reporting
+          {% endblocktrans %}
+        </p>
+        <p class="link theme-color">
+          Learn More
+        </p>
+      </a>{# .column #}
+
+      <a class="tools-cards column align-center border-all border-rounded-large pad-horz-small-at-medium pad-top-small accent-gray-bg pad-bottom-small"
+         href="{% url 'commcare' %}">
```

Comment:
@biyeun Replace commcare with ict4d, This should do since ict4d page is not yet merged. After merging PR-296, we need to merge this.

Reply:
@biyeun Done this in latest commit, So we are good to go without any changes.